api/v1/testkube.yaml

```diff
@@ -7874,6 +7875,9 @@ components:
           type: boolean
           description: whether webhooks on the executions of this test workflow are disabled
           default: false
+        runningContext:
```

Comment:
I would suggest designing a new replacement for the existing Running Context, as it's very limiting - having only type and context (= i.e. execution ID) string is limiting a lot, as we can't i.e. build the proper tree.

Reply:
ok, do we have a list of required fields?

Reply:
I'm not aware of any solid research conducted on this topic, so we rather don't have anything like that yet

Reply:
hey @jmorante-ks need some imput about what is required to store besides what we do now

Reply:
I'm agree that it would be better do design this first from UX PoV. 

@fivenp @TheBrunoLopes any thoughts? 

Currently our "context" is more about OSS. I think for now we have only one customer requirements about this: 

We know at least what Adobe is missing here: 
* they want to know who run the test - what means that it'll be the user or some API token, but we have also events taken from triggers.